dev/com.ibm.ws.jaxrs.2.1_fat_executor/fat/src/com/ibm/ws/jaxrs/executor/fat/FATSuite.java

```diff
@@ -27,6 +27,6 @@
 public class FATSuite {
     @ClassRule
     public static RepeatTests r = 
-        RepeatTests.withoutModification()
+        RepeatTests.withoutModificationInFullMode()
```

Comment:
This will be a problem on Java8 since there is no tests to run in lite mode.  You need the conditionalFullFATOnly option for without modification scenario.